pkg/restore/restore.go

```diff
@@ -816,6 +816,7 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 			addRestoreLabels(fromCluster, labels[api.RestoreNameLabel], labels[api.BackupNameLabel])
 
 			if !equality.Semantic.DeepEqual(fromCluster, obj) {
+				ctx.log.Infof("Restoring %s: %v", obj.GroupVersionKind().Kind, name)
```

Comment:
This isn't accurate. If we've made it here, it means that (1) we failed to restore the item because it already exists, and (2) that there is some difference between the backed-up copy of the item and the item that's currently in the cluster. But at this line, we don't yet know what's going to happen next. If the item is a ServiceAccount, then we'll attempt to patch it so we can merge some of the data from the backed-up copy with what's in the cluster. Otherwise, we're going to log that we couldn't restore it because it already exists and there are differences between the backed-up copy and the in-cluster version.

I think it might make more sense to do something like this:
```go
if equality.Semantic.DeepEqual(fromCluster, obj) {
  ctx.log.Infof("Skipping restore of xyz because it already exists in the cluster and is unchanged from the backed up version")
} else {
  // do the ServiceAccounts logic
}
continue
```

@skriss @rdodev wdyt about this reorg?

Reply:
Is this ok?

```go
if equality.Semantic.DeepEqual(fromCluster, obj) {
  ctx.log.Infof("Skipping restore of xyz because it already exists in the cluster and is unchanged from the backed up version")
  continue
}
switch groupResource {
case kuberesource.ServiceAccounts:
  // swap continues for breaks since a continue will occur at end of switch
default:
  // no change
}
continue
}
```

Reply:
Should be. I was trying to avoid multiple `continue` statements to try to keep it a bit less confusing. Also cc @carlisia for 👀

Reply:
A new commit is up, but I'm not marking as resolved to wait for the requested "eyes on" (or for someone else to mark).